CONTRIBUTING.md

```diff
@@ -32,31 +32,66 @@ $ ./bin/build_binary
 
 ## Development
 
-- Create a directory that will hold all the virtualenv packages and files:
+The following are instructions for setting up a development environment. Once you setup your environment, you will be able to see live changes made to the CLI.
+
+1. Create a directory that will hold all the virtualenv packages and files. You will only need to run the this command once.
+
 ```
 $ python3 -m venv venv
 ```
 
-- Enable your terminal to use those files with this command:
+2. Enable your terminal to use those files with this command. Note that this command will need to run each time you want to return to your virtual environment.
+
 ```
 $ source venv/bin/activate
 ```
 
-- Install requirements:
+2. Install requirements
+
 ```
 $ pip3 install -r requirements.txt
 ```
 
-You can now run the tests and the CLI with modifiable files!
+3. You can now run the tests and the CLI with modifiable files. 
+
+Check it out! Run the following prefix to start seeing changes
+
+```
+$ ./pkg_bin/conjur-cli <command> <subcommand>
+```
 
 ## Testing
 
 ### Unit and Integration tests
 
+To run both unit and integration tests at once, run the following:
+
 ```
 $ ./bin/test
 ```
 
+You can run specific tests by running the following:
+
+1. Drop down into a test container
+
+```
+$ /bin/test_integration -d
+```
+
+2. Under the individual test add 
+
+```
+<name-of-test-function>.<identifier>=True
+```
+
+3. Add the identifier of choice to the following command 
+
+```
+root@123456:/opt/conjur-api-python3# nose2 -v -X -A '<identifier>'
```

Comment:
can you add an example?

Reply:
added. Let me know if it still isn't clear